src/plugins/autobank/MQ2AutoBank.cpp

```diff
@@ -0,0 +1,431 @@
+/*
+ * MacroQuest: The extension platform for EverQuest
+ * Copyright (C) 2002-2022 MacroQuest Authors
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License, version 2, as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+
+#include <mq/Plugin.h>
+
+PreSetup("MQ2AutoBank");
+PLUGIN_VERSION(0.1);
+
+namespace mq {
```

Comment:
This doesn't need the namespace anymore since it is a plugin now.

Reply:
@jessebevil Just a ping to let you know this is open.

Reply:
removed the namespace. Didn't notice this here. Not used to checking github lol. 
resolved in latest commit.